README.md

```diff
@@ -93,6 +94,42 @@ and build the library with:
 npm run build-lib
 ```
 
+### Scripts For Development
+
+The following scripts can be used for development:
+
+- `npm test`: runs the unit tests of the demo application and the lib's unit tests.
+- `npm test-lib`: runs the lib's unit tests in headless mode.
+- `npm test-lib-local`: runs the lib's unit test. 
```

Comment:
Difference between `npm test` and  `npm test-lib-local` is not clear to me but may be I do not have complete idea here. 

You have explained this again in below section under `Running tests`. It feels repetitive. Is it possible to remove explanation from there and add it under `npm test` and  `npm test-lib-local`?

Reply:
I removed redundant explanations in https://github.com/dasch-swiss/dsp-ui-lib/pull/241/commits/7f1cabf58f35d3eb2a2029d760cb62ffc5b8b8b1

Reply:
> Difference between `npm test` and `npm test-lib-local` is not clear to me but may be I do not have complete idea here.
> 
> You have explained this again in below section under `Running tests`. It feels repetitive. Is it possible to remove explanation from there and add it under `npm test` and `npm test-lib-local`?

There are unit tests for the lib (`projects/dsp-ui/src`) and some unit tests for the demo app (`src/app`). `npm test` runs all the unit tests. We care mainly about the lib, but we need the demo app for the playground pages and the E2E tests.

Reply:
It is clear now. Is it possible to add this information in README as well?

Reply:
I added some more info in https://github.com/dasch-swiss/dsp-ui-lib/pull/241/commits/78a5d5e661b34b50ce0717354066cd340fad66fa

Reply:
Thanks.